python/vyos/airbag.py

```diff
@@ -157,7 +159,7 @@ def _intercepting_exceptions(_singleton=[False]):
 {instructions}
 
 When reporting problems, please include as much information as possible:
-- do not obfuscate any data (feel free to contact us privately if your 
```

Comment:
Is it just a formatting change or the message is messed up by that extra space somehow? If it's just formatting, I don't think we should mix it in the same commit with logic changes.